memtable/write_buffer_manager.cc

```diff
@@ -153,20 +237,20 @@ void WriteBufferManager::BeginWriteStall(StallInterface* wbm_stall) {
 
 // Called when memory is freed in FreeMem or the buffer size has changed.
 void WriteBufferManager::MaybeEndWriteStall() {
-  // Cannot early-exit on !enabled() because SetBufferSize(0) needs to unblock
+  // Cannot early-exit on !enabled() because SetFlushSize(0) needs to unblock
   // the writers.
   if (!allow_stall_) {
     return;
   }
 
-  if (IsStallThresholdExceeded()) {
+  if (is_stall_threshold_exceeded()) {
```

Comment:
It's not trivial, keep the style

Reply:
Why not? It's a comparison, one line basic arithmetic. The old `enabled` is named this way.